### DISCUSSION TITLE
Encryption

### DISCUSSION THREAD

**Discussion by comdora** [paolo-rossi/delphi-jose-jwt#70] [General]
Hi,

If we have cert.pem and pair.pem, is it possible to use the component?

can we encrypt from Delphi like using next script
openssl cms -sign -md sha256 -in $inputfile -signer "$ssldestination\pair.pem" | openssl cms -encrypt -aes-256-cbc -outform DER -out $outputfile "$ssldestination\cert.pem"

Regards
Bojan

**Comment by paolo-rossi** [comment]
Hello @comdora 

can you be more specific on the matter?

is this related to the JWT? AES is not among algorithms supported by the JWT standard...

Ciao,
Paolo.
